src/citrine/resources/report.py

```diff
@@ -29,13 +26,10 @@ def __init__(self, project_id: UUID, session: Session):
         self.session = session
 
     def get(self,
-            module_id: UUID = None,
-            *,
             predictor_id: Union[UUID, str] = None,
```

Comment:
I think `predictor_id` should be kept keyword-only. It might not be obvious to a caller that first argument to `get` isn't the report ID, but forcing them to name it as such should do so.